src/components/Book.jsx

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import './styles/book.css';
 
 const Book = (props) => {
-  const { author, title, id, handleDeleteProps } = props;
+  const { author, category, title, id, handleDeleteProps } = props;
   return (
-    <li className="bookContainer">
-      <p>
-        {title}
-        ::
-        {author}
-      </p>
-      <button
-        type="button"
-        data-id={id}
-        onClick={(event) => handleDeleteProps(event.target.dataset.id)}
-      >
-        delete
-      </button>
+    <li className="book-panel">
+      <div className="leftblock">
+        <span className="book-genre">{category}</span>
+        <span className="book-title">{title}</span>
+        <span className="book-author">{author}</span>
+        <span className="leftblockbuttons">
+          <button data-id={id} type="button" className="lbBtn">
+            Comments
+          </button>
```

Comment:
- [ ] You did a good job on the styles but there are a few issues you need to fix. Please use the correct color for the book buttons(comment, remove, edit) and remove the border around the book buttons, and make sure your design matches the template.

###  Your design

![image](https://user-images.githubusercontent.com/83803414/172880370-0e6fe20a-a564-454f-becb-4bf6c65d41e4.png)

### Zeplin design

![image](https://user-images.githubusercontent.com/83803414/172880336-dd05600c-0385-4e57-9eaf-98f89cbc1a8c.png)